crates/turbo-tasks/src/manager.rs

```diff
@@ -410,7 +410,7 @@ impl<B: Backend + 'static> TurboTasks<B> {
     /// Calls a native function with arguments. Resolves arguments when needed
     /// with a wrapper [Task].
     pub fn dynamic_call(&self, func: FunctionId, inputs: Vec<ConcreteTaskInput>) -> RawVc {
-        if inputs.iter().all(|i| i.is_resolved() && !i.is_nothing()) {
+        if inputs.iter().all(|i| i.is_resolved()) {
```

Comment:
is this an unrelated bug?

Reply:
Not a bug, just unnecessary. Resolving `Nothing` leads to `Nothing`. We don't have to create a resolve task to do that...